src/murmur/Server.h

```diff
@@ -169,6 +169,11 @@ class Server : public QThread {
 		// Certificate stuff, implemented partially in Cert.cpp
 	public:
 		static bool isKeyForCert(const QSslKey &key, const QSslCertificate &cert);
+		/// Attempt to load a private key in PEM format from |buf|.
+		/// If |passphrase| is non-empty, it will be used in trying to decipher the key.
```

Comment:
If it is confusing as-is, I think it should be rewritten.

How about simply

    /// If |passphrase| is non-empty, it will be used for decrypting the private key in |buf|.

?